wiki/Game_Modifiers/Nightcore/en.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
It'd be best to remove this file; #3809 is currently handling this.

Reply:
we can just merge em when the time comes. this article is mainly to make sure no information is lost from the glossary article